docs/wcs/index.rst

```diff
@@ -11,13 +11,27 @@ World Coordinate System (`astropy.wcs`)
 Introduction
 ============
 
-`astropy.wcs` contains utilities for managing World Coordinate System
-(WCS) transformations in FITS files.  These transformations map the
-pixel locations in an image to their real-world units, such as their
-position on the sky sphere.  These transformations can work both
-forward (from pixel to sky) and backward (from sky to pixel).
-
-It performs three separate classes of WCS transformations:
+World Coordinate Systems (WCSs) describe the geometric transformations
+between one set of coordinates and another. A common application is to
+map the pixels in an image onto the celestial sphere. Another common
+application is to map pixels to wavelength in a spectrum.
+
+astropy.wcs contains utilities for managing World Coordinate System
+(WCS) transformations defined in several elaborate `FITS WCS standard`_ conventions.
+These transformations work both forward (from pixel to world) and backward
+(from world to pixel).
+
+For historical reasons and to support legacy software, `astropy.wcs` maintains
+two separate application interfaces. The ``High-Level API`` should be used by
+most applications. It abstracts out the underlying object and works transparently
+with other packages which support the
+`Common Python Interface for WCS <https://zenodo.org/record/1188875#.XnpOtJNKjyI>`_,
+ allowing for a more flexible approach to the problem and avoiding the `limitations
```

Comment:
It looks like the indentation starting here is causing the html build to fail

Reply:
No problem, just changed the indentation to fix.